.sourcery-templates/DerivePartialIsos.stencil

```diff
@@ -1,5 +1,6 @@
 import ApplicativeRouter
 import Either
+import PointFreePrelude
```

Comment:
This generated file needs access to `EmailAddress` to work. Not sure how modular generated Isos should be distributed yet...

Reply:
yeah, I bet the command line tool for generating partial isos will need to take an argument for extra imports or something. I don't think SwiftSyntax can give us module names of types.